dist/milligram.css

```diff
@@ -537,11 +537,14 @@ table {
   width: 100%;
 }
 
-td,
 th {
   border-bottom: 0.1rem solid #e1e1e1;
   padding: 1.2rem 1.5rem;
 }
+tr>td {
+  border-bottom: 0.1rem solid #e1e1e1;
+  padding: 1.2rem 0 !important;
```

Comment:
Since the rule `tr>td` is more specific, do you still need `!important`?

Reply:
The reason for !important is to make it more safe if some future addition in css causes a problem, this will have a higher priority.

Reply:
That's true, but the using of `!important` is not recommended. If you take a look at milligram dist, it's never used, and it shouldn't.